modules/passkey/package.json

```
@@ -64,6 +63,7 @@
     }
   },
   "dependencies": {
+    "cbor": "^9.0.2",
     "@safe-global/safe-contracts": "^1.4.1-build.0"
```

```suggestion
    "@safe-global/safe-contracts": "^1.4.1-build.0",
    "cbor": "^9.0.2"
```

NPM automatically orders dependencies when adding/removing them with the tool, so otherwise this will generate noise.

Also make sure to NPM install after updating the package JSON, otherwise the lock file will be out of date.

---

Done 👍🏾